panels/info/cc-info-panel.c

```diff
@@ -1840,6 +1843,24 @@ cc_info_panel_init (CcInfoPanel *self)
       return;
     }
 
+  self->priv->hbox1 = WID ("hbox1");
+  self->priv->detail_vbox = WID ("detail_vbox");
+
+  if (gdk_screen_width () < 1024 || gdk_screen_height () < 600)
```

Comment:
Can you suggest a way to expose that property (technically it's not a property, but whatever)? The only feasible way would be exposing it as another function in CcShell interface, which can break potential third-party plugins.

Also, the 'small-window' property is not enough for the check that is being made here. So either we would have to change the property, or let the extra checks here,